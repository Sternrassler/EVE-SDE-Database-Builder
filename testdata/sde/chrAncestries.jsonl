{"ancestryID":1,"ancestryName":"Liberal Holders","bloodlineID":4,"description":"Wealthy Amarr nobles.","perception":0,"willpower":3,"charisma":6,"intelligence":4,"memory":2,"iconID":null,"shortDescription":"Wealthy nobles"}
{"ancestryID":2,"ancestryName":"Entrepreneurs","bloodlineID":1,"description":"Caldari business elite.","perception":9,"willpower":0,"charisma":3,"intelligence":4,"memory":3,"iconID":null,"shortDescription":"Business elite"}
{"ancestryID":11,"ancestryName":"Traders","bloodlineID":2,"description":"Minmatar merchants.","perception":7,"willpower":0,"charisma":5,"intelligence":4,"memory":3,"iconID":null,"shortDescription":"Merchants"}
