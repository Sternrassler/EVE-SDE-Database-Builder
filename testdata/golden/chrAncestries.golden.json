[
  {
    "ancestryID": 1,
    "ancestryName": "Liberal Holders",
    "bloodlineID": 4,
    "description": "Wealthy Amarr nobles.",
    "iconID": null,
    "shortDescription": "Wealthy nobles"
  },
  {
    "ancestryID": 2,
    "ancestryName": "Entrepreneurs",
    "bloodlineID": 1,
    "description": "Caldari business elite.",
    "iconID": null,
    "shortDescription": "Business elite"
  },
  {
    "ancestryID": 11,
    "ancestryName": "Traders",
    "bloodlineID": 2,
    "description": "Minmatar merchants.",
    "iconID": null,
    "shortDescription": "Merchants"
  }
]